cs/battletoken.go

```diff
@@ -84,7 +84,7 @@ func newBattleToken(num int, position BattleVector, cargoMass int, token *ShipTo
 		maxRange := 0
 		for _, slot := range token.design.Spec.WeaponSlots {
 			weapon := techFinder.GetHullComponent(slot.HullComponent)
-			bws := newBattleWeaponSlot(&battleToken, slot, weapon, hull.RangeBonus, token.design.Spec.TorpedoBonus, token.design.Spec.BeamBonus)
+			bws := newBattleWeaponSlot(&battleToken, slot, weapon, hull.RangeBonus, token.design.Spec.torpedoBonus, token.design.Spec.BeamBonus)
```

Comment:
heh, this one should be Capitalized because it's an exported field used by the UI and possibly the AI.

Reply:
I realize that now - it borking tests.